src/Bellhop.js

```diff
@@ -94,16 +94,20 @@ export class Bellhop extends BellhopEventDispatcher {
 
     // If this is not the initial connection message
     if (message.data !== 'connected') {
-      // Ignore all other message if we don't have a context
-      if (
-        this.connected &&
-        'object' === typeof message.data &&
-        message.data.type
-      ) {
-        this.trigger(message.data);
+      let data = message.data;
+      // Check to see if the data was sent as a stringified json
+      if ('string' === typeof data) {
+        try {
+          data = JSON.parse(data);
+          // eslint-disable-next-line
+        } catch (err) {}
```

Comment:
Would it be worth logging the error here just in case?
this is probably extremely unlikely to happen though.